student.rb

```diff
@@ -0,0 +1,11 @@
+class Student < Person
+  attattr_accessor :@classroom
+  def initialize(classroom, name, age, parent_permission)
+    @classroom = classroom
+    super(name, age, parent_permission)
```

Comment:
- [ ] As requested by the previous reviewer, name, and parent_permission should have default values:

  ```
  def initialize(classroom, age, name = 'Unknown', parent_permission: true)
    @classroom = classroom
    super(age, name, parent_permission: parent_permission)
  end
  ```